_posts/2017-09-08-web.md

```diff
@@ -12,9 +12,84 @@ hierarchy:
 ---
 
 
-## Web apps
+## An Introduction 
 
-TODO
+Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to provide a successful career in web development.
+
+
+#### How does the Internet work? 
+
+The Internet has become the backbone of our modern-day world, yet few individuals take the time to understand what it is, and how it works. Since it has become such a large part of our lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
```

Comment:
I would omit the "few individuals rly understand it" part. You can just say "...of our modern-day world. You should understand how it works to become a great web developer."